src/mongocrypt-ctx.c

```diff
@@ -807,10 +807,8 @@ mongocrypt_ctx_setopt_masterkey_aws (mongocrypt_ctx_t *ctx,
    bson_append_utf8 (&as_bson,
                      MONGOCRYPT_STR_AND_LEN ("provider"),
                      MONGOCRYPT_STR_AND_LEN ("aws"));
-   BSON_ASSERT (region_len <= INT_MAX);
```

Comment:
https://jira.mongodb.org/browse/SERVER-72968

Reply:
This will be removed in the next patch, and we will suppress the coverity warnings in the server codebase.